src/screens/OrdersTabScreen/OrdersTabScreen.pcss

```diff
@@ -47,6 +47,7 @@
 
     &__pagination {
       padding: var(--gap) calc(var(--gap) * 2);
+<<<<<<< HEAD
```

Comment:
please rebase carefully in future to avoid such things
and delete this line plz